mobile/library/common/http/client.cc

```diff
@@ -293,6 +312,7 @@ void Client::DirectStreamCallbacks::onComplete() {
   if (elapsed > SlowCallbackWarningThreshold) {
     ENVOY_LOG_EVENT(warn, "slow_on_complete_cb", "{}ms", elapsed.count());
   }
+  data_available_timer_.reset();
```

Comment:
I think we'll also want to disable this timer if we get a onError ,onCancel or endStream=true